libraries/api-extractor/src/ApiDefinitionReference.ts

```diff
@@ -101,10 +101,10 @@ export default class ApiDefinitionReference {
    * and deconstructs it into an IApiDefinitionReference interface object.
```

Comment:
Also document the behavior when an error is encountered - return value is undefined, error is logged via reportError delegate.